README.md

```diff
@@ -16,20 +16,31 @@ Ami app gives you flexibility to test multiple features such as :
 
 
 # Getting Started
+
 Ami app already includes all the dependencies required. To install the libraries using :
 ```
-npm install
+yarn
```

Comment:
I had better luck getting yarn to work on CI server instead of npm. So, I added `yarn.lock` to the project and use yarn on CI. 

I don't mind yarn or npm for this project. As long as it works on development environments and CI server, I am OK with either. So, this change is up for debate.

Reply:
Either of the both, `npm` or `yarn` are used to install packages. `Yarn` is better than `npm` but `npm` is widely used. I would suggest to add both the possible processes i.e `yarn install` & `npm install` with a one liner explaining each.

I can take this up once this PR is merged.